### PR TITLE
Added update to resampler

### PR DESCRIPTION
**Pull request by adityauj**
Resampler didnt handle when new resolution was lesser than native/old resolution.
Now resampler just returns old data with old resolution if ne resolution is finer/lesser than the old resolution.

### CODE REVIEW

File: .gitignore
Comment (moebiusband73) [inline .gitignore:1]: I did this for a reason :-).
If you just do config.json, any config.json will be ignored.
Maybe /config.json is correct?

File: pkg/resampler/resampler.go
Comment (moebiusband73) [inline pkg/resampler/resampler.go:1]: Same as with cc-backend:
Why does the SimpleResampler still return an error.
And why don't you just return the data if old_frequency == new_frequency

**Review by moebiusband73 — CHANGES_REQUESTED** [review]

File: pkg/resampler/resampler.go
Comment (adityauj) [reply pkg/resampler/resampler.go:1]: new_frequency <= old_frequency covers the  old_frequency == new_frequency case. Data is returned as is.

**Comment by adityauj** [comment]
Apologies for SimpleResampler error. It was never updated/maintained by me. Now I have made the function calls identical. 
Have tested both of them.

**Review by moebiusband73 — APPROVED** [review]

**Pull request closed without merging**